handlers/epoch.go

```diff
@@ -171,7 +171,7 @@ func Epoch(w http.ResponseWriter, r *http.Request) {
 		err = epochTemplate.ExecuteTemplate(w, "layout", data)
 	}
 
-	if handleTemplateError(w, r, err) != nil {
+	if handleTemplateError(w, r, "epoch.go / Epoch / Done", err) != nil {
 		return // an error has occurred and was processed
 	}
```

Comment:
Minor: You could replace all this by just `handleTemplateError(w, r, "epoch.go / Epoch / Done", err)` and get the same result.